### ISSUE TITLE
Add missing tests accessibility.spec.ts: Accessibility

### ISSUE THREAD

**Issue by kblok** [hardkoded/puppeteer-sharp#2174]
accessibility.spec.ts: Accessibility: get snapshots while the tree is re-calculated
